jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/listeners/HtmlReporter.java

```diff
@@ -115,4 +117,27 @@ public void testEnded(String host) {
 
   }
 
+  /**
+   * Allows to configure APDEX for all requests in test
+   *
+   * @param satisfied - "satisfied" threshold
+   * @param tolerated - "tolerated threshold
+   * @return HtmlReporter
+   *
+   * @since 0.57
+   */
+  public HtmlReporter adpex(Duration satisfied, Duration tolerated) {
```

Comment:
there seems to be a typo here :)

Reply:
Have you considered also to name it something like `apdexThresholds` to make it more descriptive of what is it expecting as parameters?